firefly_client/firefly_client.py

```diff
@@ -153,6 +158,32 @@ def __init__(self, url=_my_url, channel=None, html_file=_my_html_file,
         self.session = requests.Session()
         self.connect()
 
+        channel_matches = channel == os.environ['fireflyChannelLab']
```

Comment:
This line raises an exception when used inside plain Python session.

```suggestion
        channel_matches = channel == os.environ.get('fireflyChannelLab')
```
Accessing the `os.environ` dictionary this way will return None if the key does not exist (i.e. environment variable is not defined). This works in a plain Python session.